projects/packages/videopress/src/client/block-editor/blocks/video/components/playback-panel/index.tsx

```diff
@@ -88,6 +88,19 @@ export default function PlaybackPanel( { attributes, setAttributes }: VideoContr
 				) }
 			/>
 
+			<ToggleControl
+				label={ __( 'Preload Metadata', 'jetpack-videopress-pkg' ) }
+				onChange={ handleAttributeChange(
+					'preload',
+					preload === 'metadata' ? 'none' : 'metadata'
+				) }
+				checked={ preload === 'metadata' ? true : false }
```

Comment:
probably `===` is enough to set the boolean value:

```suggestion
				checked={ preload === 'metadata' }
```

Reply:
good point, gracias!